### PR TITLE
Add Investing

### PR DESCRIPTION
**Pull request by mr-karan**
https://github.com/mr-karan/awesome-investing

This is a curated list of some investment & finance related resources. The list comprises of links to books, courses, websites, podcasts about stock markets, personal finance and value investment techniques.

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:
	- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- It's the result of hard work and the best I could possibly produce.
	- Non-generated Markdown file in a GitHub repo.
	- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
	- Not a duplicate.
	- Only has awesome items. Awesome lists are curations of the best, not everything.
	- Includes a project logo/illustration whenever possible.
		- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- The image should link to the project website or any relevant website.
		- The image should be high-DPI. Set it to maximum half the width of the original image.
	- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
		- Should be placed on the right side of the readme heading.
		- Should link back to this list.
	- Has a Table of Contents section.
		- Should be named `Contents`, not `Table of Contents`.
		- Should be the first section in the list.
		- Should only have one level of sub-lists, preferably none.
	- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- If you use a license badge, it should be SVG, not PNG.
	- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- The file should be named `contributing.md`. Casing is up to you.
	- Has consistent formatting and proper spelling/grammar.
		- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
		- The description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Go to the top and read it again.

### CODE REVIEW

**Comment by mr-karan** [comment]
Hi @sindresorhus was just wondering if you got some time to take a look at this PR. 

Thanks 😄

**Comment by mr-karan** [comment]
Um, @sindresorhus just wanted to bump this thread

**Comment by legodude17** [comment]
It would be nice if the Books and Blogs had descriptions.

File: readme.md [hunk 1]
```diff
@@ -453,7 +453,7 @@
 - [OKR Methodology](https://github.com/domenicosolazzo/awesome-okr) - Goal setting & communication best practices.
 - [Leading and Managing](https://github.com/LappleApple/awesome-leading-and-managing) - Leading people and being a manager in a technology company/environment.
 - [Indie](https://github.com/mezod/awesome-indie) - Independent developer businesses.
-
+- [Investing Resources](https://github.com/mr-karan/awesome-investing) - Curated list of investment & finance related resources.
```
Comment (legodude17) [inline readme.md]: Describe the topic, not the list itself

**Comment by sindresorhus** [comment]
Hey, sorry for the lack of response, please see #1363.

**Comment by sindresorhus** [comment]
@mr-karan https://github.com/sindresorhus/awesome/pull/1091#discussion_r185953072

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/mr-karan/awesome-investing
✖ Linting

  README.md:71:3
  ✖    1:1    Missing content headers                                                  remark-lint:awesome/toc
  ✖    1:24   Invalid badge source                                                     remark-lint:awesome/badge
  ✖   17:98   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   18:110  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   19:172  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   20:83   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   21:151  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   23:135  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   24:112  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   25:136  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   26:98   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   27:136  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   28:92   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   29:86   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   30:96   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   31:106  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   32:132  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   33:94   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   34:97   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   35:137  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   36:97   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   37:143  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   38:115  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   39:90   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   40:116  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   41:78   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   42:120  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   43:110  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   44:123  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   45:94   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   46:89   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   47:188  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   48:78   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   49:92   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   50:136  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   51:166  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   52:94   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   53:174  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   54:103  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   55:140  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   56:97   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   57:102  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   58:90   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   59:129  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   60:89   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   61:82   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   62:130  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   63:117  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   64:90   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   65:111  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   66:75   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   67:116  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   68:119  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   69:91   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   70:104  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   71:3    Don’t pad link with inner spaces                                         remark-lint:no-inline-padding
  ✖   71:152  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖   75:3    Link to http://pulse.zerodha.com/ is dead                                remark-lint:no-dead-urls
  ✖   89:3    Link to http://www.gannononinvesting.com/ is dead                        remark-lint:no-dead-urls
  ✖  116:42   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖  118:100  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖  120:128  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖  122:103  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖  124:105  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖  126:123  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖  128:3    Link to https://www.coursera.org/learn/computational-investing is dead   remark-lint:no-dead-urls
  ✖  128:92   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖  130:81   List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖  132:125  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖  134:126  List item link and description must be separated with a dash             remark-lint:awesome/list-item
  ✖  136:120  List item description must end with proper punctuation                   remark-lint:awesome/list-item
  ✖  157:3    Link to http://valuepodcast.com/about is dead                            remark-lint:no-dead-urls
  ✖  159:3    Link to http://www.beyondproxy.com/tag/value-investing-podcast/ is dead  remark-lint:no-dead-urls
  ✖  185:1    Rules should use ---
```

https://github.com/sindresorhus/awesome-lint

**Comment by ironcev** [comment]
Hi @mr-karan , thanks for compiling the list!

The linter is still showing 72 errors. It's easy to fix them. To see all of them just run `awesome-lint https://github.com/mr-karan/awesome-investing`.

Another one detail from the awesome list criteria: "Entries have a description, unless the title is descriptive enough by itself. It rarely is though."

Most of the entries (books, blogs, podcasts, tools) do not have descriptions. Courses, on the other hand, are very well described.

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**